net.openra.OpenRA.json

```diff
@@ -0,0 +1,328 @@
+{
+  "app-id": "net.openra.OpenRA",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "openra-ra",
+  "rename-appdata-file": "openra-ra.appdata.xml",
+  "rename-desktop-file": "openra-ra.desktop",
+  "rename-icon": "openra-ra",
+  "finish-args": [
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--share=network",
+    "--persist=.openra"
+  ],
+  "modules": [
+    {
+      "name": "mono",
```

Comment:
There is a mono sdk packaged: https://github.com/flathub/org.freedesktop.Sdk.Extension.mono5

Reply:
Fixed